.metadata-tools/metadataanalyzer/go.mod

```diff
@@ -0,0 +1,199 @@
+module metadataanalyzer
+
+go 1.20
+
+replace github.com/dapr/components-contring => ../../
+
+require (
+	github.com/dapr/components-contrib v1.10.4-0.20230323005127-3cd727a56269
+	github.com/dapr/kit v0.0.5-0.20230321003608-efcc1af907a7
+	gopkg.in/yaml.v2 v2.4.0
+)
+
+require (
+	cloud.google.com/go v0.107.0 // indirect
+	cloud.google.com/go/compute v1.18.0 // indirect
+	cloud.google.com/go/compute/metadata v0.2.3 // indirect
+	cloud.google.com/go/datastore v1.10.0 // indirect
+	cloud.google.com/go/iam v0.11.0 // indirect
+	cloud.google.com/go/secretmanager v1.10.0 // indirect
+	github.com/Azure/azure-sdk-for-go v68.0.0+incompatible // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.5.0-beta.1 // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0-beta.4 // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos v0.3.3 // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/data/aztables v1.0.1 // indirect
```

Comment:
Yeah, auto-generating this will be great because this is covered by .gitignore  and it will be easy to forget to update these

Reply:
Yup - I will cover that via a Makefile command instead.